### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by jorgegoco**
- Changed the default state in the books reducer from an empty array to an array with a few books.
- Imported Redux Provider and the store in the index.js file.
- Displayed a list of books from the Redux store.
- Added functionality to the React component responsible for adding new books.
- Added functionality to the React component responsible for removing a book from the list of books.
- Added functionality for "Check status" button click in the "Categories" component.

### CODE REVIEW

File: src/Components/Categories.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <section className="categoriesSection">
+      <button type="button" onClick={() => { dispatch(checkStatus); }}>Check Status</button>
+    </section>
+  );
+};
```
Comment (gemmen29) [inline src/Components/Categories.js]: - [ ] Kindly fix the dispatch of the `checkStatus` action as now when I press the `Check Status` button nothing is dispatched and the redux state remain the same.

File: src/Components/Book.js [hunk 2]
```diff
@@ -1,12 +1,18 @@
 import React from 'react';
```
Comment (gemmen29) [inline src/Components/Book.js:1]: - [ ] Kindly fix this error
image
when I run your application I get this error. There is a problem in the path of the `Books` and `Categories` components.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @jorgegoco, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
Hi @jorgegoco  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights
✔️ Descriptive PR 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**